app/services/data/dashboard/get-paid-claim-count.js

```diff
@@ -0,0 +1,26 @@
+const config = require('../../../../knexfile').intweb
+const knex = require('knex')(config)
+const claimStatusEnum = require('../../../../app/constants/claim-status-enum')
+const applyFilter = require('./apply-filter')
+
+module.exports = function (filter) {
+  return applyFilter(
+      knex('Claim')
+        .count('ClaimId AS Count')
+        .where(function () {
+          // For Advance Claims
+          this.where(function () {
+            this.where({
+              'IsAdvanceClaim': true,
+              'Status': claimStatusEnum.APPROVED_ADVANCE_CLOSED.value
+            })
+          })
+          // For normal claims
+          .orWhere(function () {
+            this.where('IsAdvanceClaim', false)
+              .whereIn('Status', [ claimStatusEnum.APPROVED.value, claimStatusEnum.AUTOAPPROVED.value ])
+          })
```

Comment:
The reason these are separate is because an advance claim is considered paid when the status is APPROVED_ADVANCE_CLOSED, whereas a normal claim is considered closed when the status is APPROVED or AUTOAPPROVED.

In short, these where clauses are separate because they apply to different types of claims